sign/sign.go

```diff
@@ -99,11 +100,14 @@ func (s *Signer) SignImage(reference string) (*SignedObject, error) {
 		outputCertificate = s.options.OutputCertificatePath
 	}
 
-	err := s.SignImageInternal(context.Background(), ko, regOpts,
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
just a minute?

Reply:
Which value do you have in mind?

Reply:
no more than 5 min, but we can make this configurable as well, wdty?

Reply:
but 1 min should be enough, it should take just a few secs

Reply:
Made it configurable and default to 3 minutes.